applications/DG-Max/DGMaxProgramUtils.h

```diff
@@ -49,6 +49,22 @@ struct PointPath {
 /// string to parse \return The points on the path
 template <std::size_t DIM>
 PointPath<DIM> parsePath(const std::string& path);
+
+/// Heuristically determine the way to determine the structure description.
+/// It accepts three forms:
+///   - A single integer: the index into the predefined structures
+///   - A filename: The file should contain lines of the form 'regex,epsilon',
+///     where regex is a regular expression that matches some of the zones and
+///     epsilon is the corresponding epsilon.
+///   - An inline version of the filename using where each regex,epsilon pair is
+///     separated by a semicolon. E.g. regex1,1;regex2,12.1
+///
+/// \param input The index or file name
```

Comment:
I do not understand the API completely, it takes a string as input. So the string can be a nuber a filename, What is an inline filename? I think I do not fully understand the problem yet

Reply:
Should be clearer now, but please check.

Reply:
Yes it is thanks